client/src/plugins/error-tracking/__tests__/ErrorTrackingSpec.js

```diff
@@ -357,8 +360,89 @@ describe('<ErrorTracking>', () => {
       { key: 'plugins', value: plugins.map(({ name }) => name) }
     );
   });
+
+
+  describe('Path normalization', () => {
+
+    it('should normalize Windows paths', () => {
+
+      return expectNormalization('/C:/Users/user/test-user/Desktop/Camunda/resources/app.asar/public/');
+    });
```

Comment:
Let's make sure we always have two lines between each unit test.

Reply:
Done!